db-scripts/src/main/resources/gen-cgds-test-schema.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+CGDS_SCHEMA_FILE=$(find . -type f -name "*cgds.sql" | grep -v target)
```

Comment:
I *think* it is ok to rely on maven setting the working directory to be that of the pom file, so this should work even if the actual working directory was elsewhere.
Here the cgds-test.sql file is created in the src tree ... rather than target.
After building using this script, does it result in an untracked file db-scripts/src/main/resources/cgds-test.sql? Maybe it would  be safer to instead use the target tree for constructing the cgds-test.sql ... that way it will be ignored and not accidentally checked in to the repo.

Reply:
@sheridancbio I added `cgds-test.sql` to the list of files to be ignored in `.gitignore`